cmd/antrea-agent/options.go

```diff
@@ -155,6 +155,11 @@ func (o *Options) validate(args []string) error {
 		return fmt.Errorf("nodeType %s requires feature gate ExternalNode to be enabled", o.config.NodeType)
 	}
 
+	// validate FqdnCacheMinTTL
```

Comment:
no need to have the comment as the code is obvious and the comment doesn't provide more information